spec/bowlingSpec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+describe('Bowling', function(){
+  var bowling;
+
+  beforeEach( function() {
+    bowling = new Bowling();
+  })
+
+  describe('#score', function(){
```

Comment:
Very strong 'feature testing' here through thorough unit-testing of the Bowling class